src/astro/utils/dataframe.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
```

Comment:
@utkarsharma2, we've been trying to move out of the utils package 🙈 
if we handle the change of capitalization from within the database class, this method could live there.

Reply:
As per discussion with @tatiana since the logic is specific to dataframes it makes sense to keep them in Utils.